index.html

```diff
@@ -48,13 +63,149 @@ <h3>LET'S CONNECT</h3>
             </div>
         </section>
 
+        <section class="desktop-container">
+           <div class="mobile-version">
+            <div class="mobile-image">
+              <img src="desktop4.svg" alt="" class="card-img-img">
+            </div>
+            <div class="mobile-section-div">
+              <h1>Tonic</h1>
+              <div class="canopy-div">
+                <p>Canopy</p>
+                <img src="Counter.svg" alt="" />
+                <p>Backend Dev</p>
+                <img src="Counter.svg" alt="" />
+                <p>2015</p>
+              </div>
+             <p class="writeup-div">A daily selection of privately personalized reads; no accounts or sign ups required</p>
+             <button class="desktop-btn">HTML</button>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
```

Comment:
- [ ] I kindly suggest that the `See Project` button should takes you to the Live Demo of the project or to the GitHub repository